src/openms/source/FORMAT/MascotRemoteQuery.cpp

```diff
@@ -181,54 +182,54 @@ namespace OpenMS
     // content
     QByteArray loginbytes;
     QString boundary_string("--" + boundary + "\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"username\"\r\n");
     loginbytes.append("\r\n");
-    loginbytes.append(String(param_.getValue("username").toString()).toQString());
+    loginbytes.append(String(param_.getValue("username").toString()).toQString().toUtf8());
     loginbytes.append("\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"password\"\r\n");
     loginbytes.append("\r\n");
-    loginbytes.append(String(param_.getValue("password").toString()).toQString());
+    loginbytes.append(String(param_.getValue("password").toString()).toQString().toUtf8());
     loginbytes.append("\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"submit\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("Login\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"referer\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"display\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("nothing\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"savecookie\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("1\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"action\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("login\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"userid\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("\r\n");
-    loginbytes.append(boundary_string);
+    loginbytes.append(boundary_string.toUtf8());
     loginbytes.append("Content-Disposition: ");
     loginbytes.append("form-data; name=\"onerrdisplay\"\r\n");
     loginbytes.append("\r\n");
     loginbytes.append("login_prompt\r\n");
-    loginbytes.append("--" + boundary + "--\r\n");
+    loginbytes.append(("--" + boundary + "--\r\n").toUtf8());
```

Comment:
this effectively is a `boundary_string`?

```
loginbytes.append(boundary_string.toUtf8());`
```

Reply:
there is small difference 
```boundary_string = "--" + boundary + "\r\n"
```
vs 
```
"--" + boundary + "--\r\n"
```

Reply:
`--` at the end

Reply:
ah.. my bad. good catch!